qutebrowser/utils/version.py

```diff
@@ -123,6 +123,15 @@ def distribution():
                             id=dist_id)
 
 
+def sandboxed():
+    """Check whether the environment is a sandbox, i.e. has restricted access to the host system.
+
+    Return:
+        True if the environment is a sandbox.
+    """
```

Comment:
Can probably drop that docstring, as the function is simple enough to not need any explanation.